test/tools/test_tools_target.py

```diff
@@ -1,10 +1,54 @@
 """Tests for targets: Channel & User"""
 from __future__ import annotations
 
+import pytest
+
 from sopel import plugin
 from sopel.tools import Identifier, target
 
 
+def test_user():
+    nick = Identifier('Brian')
+    username = 'brian'
+    host = 'example.com'
+    user = target.User(nick, username, host)
+
+    assert user.nick == nick
+    assert user.user == username
+    assert user.host == host
+    assert user.realname is None
+    assert user.channels == {}
+    assert user.account is None
+    assert user.away is None
+    assert user.hostmask == '%s!%s@%s' % (nick, username, host)
```

Comment:
I neglected to include this specific check in my `test_user()` case from #2448, so you can just add it + the other cases below when rebasing. 😺